test_pull_requests

```diff
@@ -1631,6 +1630,48 @@ popd
         end
       end
     end
+
+    private
+
+    def http_get(uri)
```

Comment:
if we have this we should mirror the ones we have for GitHub: `get` `delete` etc